### ISSUE TITLE
Hovering "mapillaryimagevieworoverlay__overlay__rect" changes its size

### ISSUE THREAD

**Issue by de-vries** [mapillary/mapillary_issues#781]
Here is a sign:
![4](https://cloud.githubusercontent.com/assets/1449609/6503842/fd3ad310-c330-11e4-89b6-ac0557197aec.png)

Here is the same sign hovered:
![5](https://cloud.githubusercontent.com/assets/1449609/6503846/044564cc-c331-11e4-94fb-69abf203bd78.png)
*It's bigger because the border in hover state gets added to the size*

Here is the same sign with 
`.mapillaryimagevieworoverlay__overlay__rect {
    border: 1px solid rgba(255, 0, 0, 0.7);
}` added to the css. My previous solution.
![6](https://cloud.githubusercontent.com/assets/1449609/6503887/6836ed7a-c331-11e4-8afe-bdc3e28a7612.png)
*if I make the overlay smaller I also change (degrade) the data of this sign*

If you also add `box-sizing: border-box;` you'll get this: 
![3](https://cloud.githubusercontent.com/assets/1449609/6503945/fd1107f0-c331-11e4-9fdf-9e25557de9b5.png)
*Same size as it is now, but with a border*

TL;DR: add this css:

     .mapillaryimagevieworoverlay__overlay__rect {
       box-sizing: border-box; 
       border: 1px solid rgba(255, 0, 0, 0.7);
     }

Note:
If you only use `box-sizing: border-box`, that will solve the jumpiness for the overlay_rect, but not for the icons next to it.

**Comment by knikel** [comment]
Fixed, thank you for the input!
